Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/OS - Default Windows 10 Update Ring.json

```diff
@@ -25,7 +25,7 @@
   "deadlineForFeatureUpdatesInDays": 5,
   "deadlineForQualityUpdatesInDays": 3,
   "deadlineGracePeriodInDays": 1,
-  "description": "Default Windows Update configuration for devices.",
+  "description": "What is it: Defines the default Windows Update configuration for Intune managed devices. Applies to all corporate devices except those in the Insiders or Preview update rings. \n Simeon Notes: This is the catch all Windows Update Ring that includes all devices not included in other rings. Simeon's baseline delays feature updates for 30 days and quality updates for 14 days after released by Microsoft. Once the deferral period has expired for the device, users have 3 days to restart (if required). Also, unattended updates are only be applied outside the working hours of 5am to 10pm. \n Learn More: https://techcommunity.microsoft.com/t5/windows-it-pro-blog/tactical-considerations-for-creating-windows-deployment-rings/ba-p/746979",
```

Comment:
**What does this do?**
Defines the default Windows Update configuration for Intune-managed devices. The baseline delays feature updates for 30 days and quality updates for 14 days after released by Microsoft. Once the deferral period has expired for the device, users have 3 days to restart (if required). Unattended updates are only applied outside the working hours of 5am to 10pm. The Windows Update configuration applies to all corporate devices except those in the Insiders or Preview update rings.

**Why should you use this?**
If you want to ensure that Windows updates have been tested by the Insiders and Preview rings before applying to all corporate devices. 

**What is the end-user impact?**
Corporate devices outside of the Insiders or Preview rings will receive Windows feature updates 30 days after being released and quality updates 14 days after being released by Microsoft.  

**Learn more**
[Tactical considerations for creating Windows deployment rings](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/tactical-considerations-for-creating-windows-deployment-rings/ba-p/746979)

Reply:
Why should you use this, should focus more on to ensure your devices are kept up to date with the latest Windows updates.